src/DynamoCoreWpf/Views/Core/AnnotationView.xaml.cs

```diff
@@ -165,10 +173,7 @@ private void AnnotationView_OnMouseLeftButtonDown(object sender, MouseButtonEven
 
         private void AnnotationView_OnMouseRightButtonDown(object sender, MouseButtonEventArgs e)
         {
-            DynamoSelection.Instance.ClearSelection();
-            System.Guid annotationGuid = this.ViewModel.AnnotationModel.GUID;
-            ViewModel.WorkspaceViewModel.DynamoViewModel.ExecuteCommand(
-               new DynCmd.SelectModelCommand(annotationGuid, Keyboard.Modifiers.AsDynamoType()));
```

Comment:
I was just looking at the original commit by @ramramps who introduced this. Trying to think about any case we may miss by removing this. https://github.com/DynamoDS/Dynamo/commit/360fdcb9f764f2758fd7c5cb1719aaebe44d2a94

Reply:
Thinking about this a bit, unselect on right click does seem a bit too aggressive. So looks good to me

Reply:
I thought this too, that is why I tested all the options available on the right click, and out of them 2 were making use of the unselect on right click code, so I handled each of those cases, which were delete and ungroup.